CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Change Log
+
+All notable changes to the Pony compiler and standard library will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a CHANGELOG](http://keepachangelog.com/).
```

Comment:
We should replace "Pony compiler and standard library" here with Corral. I'll add that to the issue for the changelog tool's "new" command

Reply:
Done.